charts/cosi/Chart.yaml

```diff
@@ -27,7 +27,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.2.0
```

Comment:
Should version be 0.1.1 for a patch release?

Reply:
Absolutely !